auth-web/src/stores/staff.ts

```diff
@@ -25,6 +25,7 @@ export const useStaffStore = defineStore('staff', () => {
     accountUnderReviewAdminContact: {} as Contact,
     accountUnderReviewAffidavitInfo: {} as AffidavitInformation,
     activeStaffOrgs: [] as Organization[],
+    involuntaryDissolutionBatch: {} as InvoluntaryDissolutionIF,
```

Comment:
On the plus side, `involuntaryDissolutionBatch.batchSize` won't throw an error. On the down side, you can't check if `involuntaryDissolutionBatch` is null. Depends how you use it, of course, but it's something to watch out for.

An alternative is to initialize the properties here as well.
```
involuntaryDissolutionBatch: {
  batchSize: null // or NaN
  onHold: null // or False
}
```

Reply:
OK, will keep this in mind Sev.

I really like your suggested change. It depends on what I'll be doing here next ticket once I tie everything together. Are you OK with me keeping it as is for now?

Reply:
You bet. Resolving.